test/boost/object_storage_upload_test.cc

```diff
@@ -6,8 +6,10 @@
  * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
  */
 
+#include "seastar/core/sstring.hh"
```

Comment:
Seastar include should use <> since they are "system headers" from our point of view